fs/expand_glob_test.ts

```diff
@@ -61,7 +61,10 @@ Deno.test("expandGlobWildcard", async function () {
 
 Deno.test("expandGlobTrailingSeparator", async function () {
   const options = EG_OPTIONS;
-  assertEquals(await expandGlobArray("*/", options), ["a[b]c", "subdir"]);
+  assertEquals(await expandGlobArray("*/", options), [
+    "a[b]c",
+    "subdir",
+  ]);
```

Comment:
I guess this is just a formatting change, maybe cleaner to omit this from the PR so it's easier to review?